<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Tempus Sans ITC</family>
           <pointsize>-1</pointsize>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.8);
    font-size: 20px;
  
    text-transform: uppercase;
    background: rgba(135, 206, 250, 0.5);
    padding: 5px;
}</string>
         </property>
         <property name="text">
          <string>my host                </string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditForMyHost"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Tempus Sans ITC</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.8);
    font-size: 20px;
    text-transform: uppercase;
    background: rgba(135, 206, 250, 0.5);
    padding: 5px; 
}</string>
         </property>
         <property name="text">
          <string>my port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxForMyPort">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Tempus Sans ITC</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.8);
    font-size: 20px;

    text-transform: uppercase;
    background: rgba(135, 206, 250, 0.5);
    padding: 5px; 
}</string>
         </property>
         <property name="text">
          <string>other host         </string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditForOtherHost"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Tempus Sans ITC</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: rgba(255, 255, 255, 0.8);
    font-size: 20px;
   
    text-transform: uppercase;
    background: rgba(135, 206, 250, 0.5); 
    padding: 5px; 
}</string>
         </property>
         <property name="text">
          <string>other port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxForOtherPort">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="saveBotton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 20px;
    background-color: rgba(135, 206, 250, 0.5); 
    padding: 10px 20px;
    color: #FFFFFF;
    font-size: 16px;
    border: 2px solid rgba(70, 130, 180, 0.5);
    transition: background-color 0.3s;
}

QPushButton:hover {
    background-color: rgba(135, 206, 250, 0.7); 
}

QPushButton:pressed {
    background-color: rgba(70, 130, 180, 0.7); 
    border: 2px solid rgba(70, 130, 180, 0.7);
}
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBotton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 20px;
    background-color: rgba(135, 206, 250, 0.5); 
    padding: 10px 20px;
    color: #FFFFFF;
    font-size: 16px;
    border: 2px solid rgba(70, 130, 180, 0.5);
    transition: background-color 0.3s;
}

QPushButton:hover {
    background-color: rgba(135, 206, 250, 0.7); 
}

QPushButton:pressed {
    background-color: rgba(70, 130, 180, 0.7); 
    border: 2px solid rgba(70, 130, 180, 0.7);
}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
